x/tx/signing/aminojson/encoder.go

```diff
@@ -166,24 +165,15 @@ func moduleAccountEncoder(_ *Encoder, msg protoreflect.Message, w io.Writer) err
 // also see:
 // https://github.com/cosmos/cosmos-sdk/blob/b49f948b36bc991db5be431607b475633aed697e/proto/cosmos/crypto/multisig/keys.proto#L15/
 func thresholdStringEncoder(enc *Encoder, msg protoreflect.Message, w io.Writer) error {
-	pk, ok := msg.Interface().(*multisig.LegacyAminoPubKey)
-	if !ok {
-		return errors.New("thresholdStringEncoder: msg not a multisig.LegacyAminoPubKey")
-	}
-	_, err := fmt.Fprintf(w, `{"threshold":"%d","pubkeys":`, pk.Threshold)
+	fields := msg.Descriptor().Fields()
```

Comment:
I thunk @kocubinski is already fixing this in here: https://github.com/cosmos/cosmos-sdk/pull/21825